applications/dashboard/modules/class.tablesummarymodule.php

```diff
@@ -0,0 +1,162 @@
+
+        if ($columnClass) {
+            $class .= ' '.$columnClass;
+        }
+        $attributes['class'] = $class;
+        $this->columns[$key] = [
+            'key' => $key,
+            'name' => $name,
+            'attributes' => $attributes,
+            'column-class' => $columnClass
+        ];
+
+        return $this;
+    }
+
+    /**
+     * Add a column to the table if it satisfies the $isAllowed condition.
+     *
+     * @param bool|string|array $isAllowed Either a boolean to indicate whether to actually add the item
+     * or a permission string or array of permission strings (full match) to check.
+     * @param string $key The key name of the column.
+     * @param string $name The display-name of the column.
+     * @param array $attributes The attributes for the th cell.
+     * @param string $columnClass The css class to propagate to every cell in this column.
+     * @return TableSummaryModule $this
+     */
+    public function addColumnIf($isAllowed, $key, $name = '', $attributes = [], $columnClass = '') {
+        if (!$this->allowed($isAllowed)) {
+            return $this;
+        }
+        return $this->addColumn($key, $name, $attributes, $columnClass);
+    }
+
+    /**
+     * Adds a row to the table. Constrains our data to the columns we've added.
+     *
+     * @param array $row An array representing a row where the key matches a column key and the value is either an
+     * HTML-formatted string or an object with a toString() method. (Something echo-able.)
+     * @param string $key The key for the row. Nothing to do with the column key. Here for expansion, like in
+     * case we want to sort items eventually.
+     * @param array $attributes The attributes on the tr element.
+     * @return TableSummaryModule $this
+     */
+    public function addRow($row = [], $key = '', $attributes = []) {
+        if (!$key) {
+            $key = randomString(8);
+        } else {
+            $key = slugify($key);
+        }
+
+        // Match up data with our columns
+        $cells = array_intersect_key($row, $this->columns);
+        $cellData = [];
+
+        foreach ($cells as $cellKey => $cell) {
+            $cellData[$cellKey]['data'] = $cell;
+        }
+        $this->rows[$key]['cells'] = $cellData;
+        $this->rows[$key]['attributes'] = $attributes;
+
+        return $this;
+    }
+
+    /**
+     * Prepares the media item for rendering. Adds column-type css classes to cells in the table.
+     *
+     * @return bool Whether to render the module.
+     */
+    public function prepare() {
+        // Add css classes to cells
+        foreach ($this->rows as $rowKey => $row) {
+            foreach ($row['cells'] as $key => $value) {
+                if (isset($this->columns[$key]['column-class'])) {
+                    $this->rows[$rowKey]['cells'][$key]['attributes']['class']
+                        = self::CSS_PREFIX.'-cell-'.$key.' '.$this->columns[$key]['column-class'];
```

Comment:
Having the assignment on a separate line is bumming me out.  Maybe build this value on a separate line, assign its value to a variable and set that in your array?

```php
$rowClass = self::CSS_PREFIX.'-cell-'.$key.' '.$this->columns[$key]['column-class'];
$this->rows[$rowKey]['cells'][$key]['attributes']['class'] = $rowClass;
```